### DISCUSSION TITLE
Core-metadata communication with other services

### DISCUSSION THREAD

**Discussion by neerajasjawali** [edgexfoundry/community#116] [Q&A] [answered]
Hi All, 

Is it written somewhere stating that only support-notification and core-metadata can communicate?
Because only in the core-metadata configuration.toml I could see the Notifications section. As shown below
```
[Notifications]        
PostDeviceChanges = true         
Slug ='device-change-'
Content = "Device update: "
Sender = "core-metadata"
Description = "Metadata device notice"
Label = "metadata"     
```

Can I add support-notifications as clients and Notifications section in core-data or any other services? 

Thanks in advance

**Comment by cloudxxx8** [comment, marked as answer]
No, currently only core-metadata provides this mechanism.
This mechanism is going to be replaced by Device System Events in v3.

**Reply by neerajasjawali** [reply]
> No, currently only core-metadata provides this mechanism. This mechanism is going to be replaced by Device System Events in v3.

Appreciate your help. Thank you @cloudxxx8
